src/scss/base/_typography.scss

```diff
@@ -78,8 +78,8 @@
     h1,
     .alpha {
         @include font-size(heading-xl, true, narrow);
-        margin-top: $baseline * 2;
-        margin-bottom: $baseline;
+        margin-top: spacing(d) * 2;
```

Comment:
Could use `spacing(f)` here and get rid of the `* 2`

Reply:
Fixed.